src/test/java/net/bramp/ffmpeg/FFprobeTest.java

```diff
@@ -160,4 +163,12 @@ public void testProbeSideDataList() throws IOException {
             "\n00000000:            0      -65536           0\n00000001:        65536           0           0\n00000002:            0           0  1073741824\n"));
     assertThat(info.getStreams().get(0).side_data_list[0].rotation, is(90));
   }
+
+  @Test
+  public void testChaptersWithLongIds() throws IOException {
+    FFmpegProbeResult info = ffprobe.probe(Samples.chapters_with_long_id);
+
+    assertThat(info.getChapters().get(0).id, is(6613449456311024506L));
+    assertThat(info.getChapters().get(1).id, is(-4433436293284298339L));
+  }
```

Comment:
Are chapter IDs allowed to be negative?

Reply:
Very good question, I just checked in the ffmpeg source.
They use int64_t, which is 64bit signed, so type long and negative values are supported.

I'm not sure, if ffmpeg itself will produce negative numbers, might be compatibility with other software.